zipline/lib/adjust.pyd

```diff
@@ -0,0 +1,607 @@
+cdef class Adjustment:
```

Comment:
this was committed incorrectly, I have a pxd also. this makes me think it is not needed since the tests ran.